testing/test_crossbrowsertesting.py

```diff
@@ -80,4 +80,4 @@ def test_invalid_credentials_file(failure, monkeypatch, tmpdir):
     config.write("[credentials]\nusername=foo\nkey=bar")
     out = failure("--capability", "browser_api_name", "FF46")
     messages = ["missing auth", "basic auth failed"]
-    assert any(message in out for message in messages)
+    assert any(message in out.lower() for message in messages)
```

Comment:
@nicoddemus I'll remove these tests instead. They're basically testing that CBT returns an error with incorrect credentials.

A case we don't handle explicit anyway. So I don't really see the point of them. 🤔 

I even think it was an earlier version of me that implemented them (or at least some of them).